keyboards/atreus/feather/rules.mk

```diff
@@ -0,0 +1,35 @@
+# MCU name
+MCU = atmega32u4
+F_CPU = 8000000
+
+BOOTMAGIC_ENABLE = no         # Virtual DIP switch configuration(+1000)
+MOUSEKEY_ENABLE = yes         # Mouse keys(+4700)
+# EXTRAKEY_ENABLE = yes       # Audio control and System control(+450)
+CONSOLE_ENABLE = no           # Console for debug(+400)
+COMMAND_ENABLE = no           # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+# SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+# NKRO_ENABLE = yes           # USB Nkey Rollover
+# BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality on B7 by default
+# RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+# MIDI_ENABLE = no            # MIDI support (+2400 to 4200, depending on config)
+UNICODE_ENABLE = yes          # Unicode
+
+BLUETOOTH_ENABLE = yes
+BLUETOOTH = AdafruitBLE
+OUTPUT_AUTO_ENABLE = yes
+
+# AUDIO_ENABLE = no           # Audio output on port C6
+# FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+# HD44780_ENABLE = no 		# Enable support for HD44780 based LCDs (+400)
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = caterina
```

Comment:
```suggestion
BOOTLOADER = caterina

# Build Options
#   change yes to no to disable
#
BLUETOOTH = AdafruitBLE
```

Reply:
Is this trying to get this to be a "base" so that people can add the normal options if they wish?  It seems like we would at least want `OUTPUT_AUTO_ENABLE = yes`.  Maybe `MOUSEKEY_ENABLE = yes         # Mouse keys(+4700)` would be required to use the clash keyboard layout?  Are the rest defaults that don't actually affect output code?  I know that mousekey and output_auto are usually false as well.

Reply:
OUTPUT_AUTO_ENABLE isnt a valid `rules.mk` option, it would have to go in a `config.h` for it to have any effect. As for mousekeys, it is enabled at the parent level <https://github.com/qmk/qmk_firmware/blob/master/keyboards/atreus/rules.mk#L5>.

This new keyboard revision should only set the minimal options required for it to function, as per the rest of the other revisions.